openpype/plugins/publish/collect_scene_loaded_versions.py

```diff
@@ -0,0 +1,61 @@
+
+import pyblish.api
+
+
+class CollectSceneLoadedVersions(pyblish.api.ContextPlugin):
+
+    order = pyblish.api.CollectorOrder + 0.0001
+    label = "Collect Versions Loaded in Scene"
+    hosts = [
+        "aftereffects",
+        "blender",
+        "celaction",
+        "fusion",
+        "harmony",
+        "hiero",
+        "houdini",
+        "maya",
+        "nuke",
+        "photoshop",
+        "resolve",
+        "tvpaint"
+    ]
+
+    def process(self, context):
+        from avalon import api, io
+
+        current_file = context.data.get("currentFile")
```

Comment:
I don't think this is required...

Reply:
Oh yeah, you are right. The code below and the other plugin still could work as expect even without this check.